platform/security/platform-security-session/pom.xml

```diff
@@ -45,6 +50,9 @@
                         </Import-Package>
                         <Import-Package/>
                         <Export-Package/>
+                        <Embed-Dependency>
+                            bcprov-jdk15on
```

Comment:
As soon as [DDF-2051](https://github.com/codice/ddf/pull/709) is merged this embed should be removed.